my_list.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './enum_module'
+
+class MyList
+  def initialize(*arr)
+    @list = arr
+  end
+
+  include MyEnum
```

Comment:
- [ ] [OPTIONAL] It is a better practice to include your imported module at the top of the class just before your `initialize` method